lib/utils/schema/inspec.exec.full.json

```diff
@@ -0,0 +1 @@
+{"type":"object","additionalProperties":false,"properties":{"profiles":{"type":"array","items":{"type":"object","additionalProperties":false,"properties":{"name":{"type":"string"},"version":{"type":"string","optional":true},"title":{"type":"string","optional":true},"maintainer":{"type":"string","optional":true},"copyright":{"type":"string","optional":true},"copyright_email":{"type":"string","optional":true},"license":{"type":"string","optional":true},"summary":{"type":"string","optional":true},"supports":{"type":"array","items":{"type":"object","additionalProperties":false,"properties":{"os-family":{"type":"string","optional":true}}},"optional":true},"controls":{"type":"array","items":{"type":"object","additionalProperties":false,"properties":{"id":{"type":"string"},"title":{"type":["string","null"]},"desc":{"type":["string","null"]},"impact":{"type":"number"},"refs":{"type":"array","items":{"type":"object","additionalProperties":false,"properties":{"ref":{"type":"string"},"uri":{"type":"string","optional":true},"url":{"type":"string","optional":true}}}},"tags":{"type":"object"},"code":{"type":"string"},"source_location":{"type":"object","properties":{"ref":{"type":"string"},"line":{"type":"number"}}},"results":{"type":"array","items":{"type":"object","additionalProperties":false,"properties":{"status":{"type":"string"},"code_desc":{"type":"string"},"run_time":{"type":"number"},"start_time":{"type":"string"},"skip_message":{"type":"string","optional":true},"resource":{"type":"string","optional":true}}}}}}},"groups":{"type":"array","items":{"type":"object","additionalProperties":false,"properties":{"id":{"type":"string"},"title":{"type":"string","optional":true},"controls":{"type":"array","items":{"type":"string"}}}}},"attributes":{"type":"array"}}}},"statistics":{"type":"object","additionalProperties":false,"properties":{"duration":{"type":"number"}}},"version":{"type":"string"},"controls":"array","other_checks":"array"}}
```

Comment:
Is there any harm in prettyifying this to make it easier for people to see? I realize the output out of the tool isn't "pretty" (and perhaps we should change that) but for purposes of comprehension, it'd be good to see this formatted nicely for humans.